### PR TITLE
feat: Add descriptions to schema objects

### PR DESCRIPTION
**Pull request by wolfy1339**
There's some little property updates mixed in there as well, shouldn't hurt.

### CODE REVIEW

**Comment by wolfy1339** [comment]
@G-Rath Since we scrape the GitHub Webhook docs and cache them, could we use them to automatically add/update descriptions, or add new properties to the schemas?

**Comment by wolfy1339** [comment]
I looked into getting properties from the docs HTML in order to add them into `index.json`, and this is what I've got up to now: https://github.com/octokit/webhooks/commit/e8f402160516083cce03c30a9e6bd5ac32fa05af

The format needs to be tweaked a little bit, but it's a step in the right direction

**Comment by wolfy1339** [comment]
@oscard0m @G-Rath @gr2m If one of you could review the PR, it would be appreciated

**Review by gr2m — APPROVED** [review]

File: payload-schemas/schemas/check_run/completed.schema.json [hunk 1]
```diff
@@ -37,21 +38,20 @@
         "status": {
           "type": "string",
           "enum": ["completed"],
-          "description": "The phase of the lifecycle that the check is currently in."
+          "description": "The current status of the check run. Can be `queued`, `in_progress`, or `completed`."
         },
         "conclusion": {
-          "type": ["string", "null"],
+          "type": "string",
```
[the remark this answers is not in the document]
Comment (wolfy1339) [reply payload-schemas/schemas/check_run/completed.schema.json:44]: I'll revert them in this PR, and submit them seperately

**Comment by G-Rath** [comment]
@wolfy1339 I forgot that adding a description to schemas causes `json-schema-to-typescript` to extract object schemas out into interfaces.

This means our schema now is littered with interfaces like `User1`, `User2`, `User3`, `Team1`, `Team2`, etc.

The bulk of the descriptions you've added look fine, but I'd like to revert the ones that result in the new interfaces for now as the duplication is very confusing; are you ok for me to do that?

**Pull request merged**